devlink_linux.go

```diff
@@ -757,6 +764,11 @@ func (h *Handle) DevlinkPortFnSet(Socket string, Bus string, Device string, Port
 	if FnAttrs.StateValid {
 		fnAttr.AddRtAttr(DEVLINK_PORT_FN_ATTR_STATE, nl.Uint8Attr(FnAttrs.FnAttrs.State))
 	}
+
+	if FnAttrs.TrustValid {
```

Comment:
There should be a check (at the top of this method):

if FnAttrs.TrustValid==true and Socket!="mlxdevm" then fail

as we cannot set trust with any other socket ATM

Reply:
Sounds good @adrianchiris  ; @l8huang could you lok at adding this change, probably also when we call port.parseAttributes.

Reply:
updated, PTAL